internal/buildscripts/packaging/msi/splunk-otel-collector.wxs

```diff
@@ -67,8 +67,11 @@
 
       <!-- Copy the default agent config file to ProgramData if it does not already exist -->
       <CustomAction Id="CopyConfig" ExeCommand="xcopy /y &quot;[INSTALLDIR]agent_config.yaml&quot; &quot;[CommonAppDataFolder]Splunk\OpenTelemetry Collector\agent_config.yaml*&quot;" Directory="INSTALLDIR" Impersonate="no" Execute="deferred" Return="check" />
+      <!-- Copy the JMX Metric Gatherer JAR to opt if it does not already exist -->
+      <CustomAction Id="CopyJMXJAR" ExeCommand="xcopy /y &quot;[INSTALLDIR]opentelemetry-java-contrib-jmx-metrics.jar&quot; &quot;[WindowsVolume]\opt\opentelemetry-java-contrib-jmx-metrics.jar*&quot;" Directory="INSTALLDIR" Impersonate="no" Execute="deferred" Return="check" />
       <InstallExecuteSequence>
          <Custom Action="CopyConfig" After="InstallFiles">NOT CONFIG_FILE_EXISTS AND NOT Installed</Custom>
+         <Custom Action="CopyJMXJAR" After="InstallFiles">UPGRADINGPRODUCTCODE OR NOT Installed</Custom>
```

Comment:
Would it be better to install the jar file directly to `\opt`?  Otherwise, I think we should add a check, similar to https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/splunk-otel-collector.wxs#L58, to copy the jar file if it does not exist.

Reply:
I've changed it to be installed automatically to `/opt`, in `./msi-build/build.sh`. I've removed this copy command.